examples/simple/variables.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "name" {
+  type        = string
+  description = "Name of the router"
+}
+
+variable "network" {
+  type        = string
+  description = "A reference to the network to which this router belongs"
+}
+
+variable "project" {
+  type        = string
+  description = "The GCP project ID"
+}
+
+variable "region" {
+  type        = string
+  description = "The GCP region"
```

Comment:
Set a default value

Reply:
Hardcoded now as that is preferred for CFT examples over default vars.